src/Websites/Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -122,6 +122,9 @@ public class SetAzureWebAppSlotCmdlet : WebAppSlotBaseCmdlet
         [Parameter(ParameterSetName = ParameterSet1Name, Mandatory = false, HelpMessage = "Azure Storage to mount inside a Web App for Container. Use New-AzWebAppAzureStoragePath to create it")]
         public WebAppAzureStoragePath[] AzureStoragePath { get; set; }
 
+        [Parameter(HelpMessage = "By default FTP is all allowed")]
```

Comment:
```suggestion
        [Parameter(HelpMessage = "Set the Ftps state value for an app. Default value is 'AllAllowed'")]
```

Reply:
The Helpmessage has been updated as per request..